src/Components/Layout.tsx

```diff
@@ -0,0 +1,15 @@
+import { FC } from 'react';
+import { Outlet } from 'react-router-dom';
+import { Navigation } from './Navigation';
+
+export const Layout: FC = () => (
+  <>
+    <Navigation />
+
+    <div className="section">
+      <div className="container">
+        <Outlet />
+      </div>
+    </div>
+  </>
```

Comment:
You don't have to use this container in layout when you use it on pages

```suggestion
  <>
    <Navigation />

    <div className="section">
      <Outlet />
    </div>
  </>
```